components/automate-ui/src/app/modules/license/license-apply/license-apply.component.ts

```diff
@@ -85,7 +77,7 @@ export class LicenseApplyComponent implements AfterViewInit, OnChanges {
 
     this.clearErrors();
     this.applyingLicense = true; // round-trip initiating...
-    this.store.dispatch(new ApplyLicense({ license: license.trim() }));
+    this.licenseFacade.dispatch(new ApplyLicense({ license: license.trim() }));
```

Comment:
couldn't we move the actions into the facade? so `license-apply` doesn't even need to import the `ApplyLicense` action, it can just call `licenseFacade.applyLicense(license)`

Reply:
I agree;
see example in components/automate-ui/src/app/entities/projects/project.service.ts

Reply:
However, after conversation with @tarablack01 , I think it is OK to postpone this change to a follow-up PR if it is going to touch more files.